.github/assets/updatecli.yml

```diff
@@ -0,0 +1,47 @@
+name: Update uv lock dependencies
```

Comment:
folder name `updatecli/updatecli.d`, that is the default name for pipelines and `updatecli/values.d` for config things. See https://github.com/updatecli/updatecli/tree/main/updatecli